healthScore/views.py

```diff
@@ -269,21 +270,21 @@ def registration(request):
             context["error_message"] = (
                 "An account already exists for this email address. Please log in."
             )
-            return render(request, "registration.html", context, status=500)
+            return render(request, "registration.html", context)
```

Comment:
Ohhhh. Now I get why the status was 200 for the test login failed test cases as well. Could you please add the status codes for the return render() functions and assign those status codes in the test cases?
By default, they are 200